docs/release.md

```diff
@@ -176,17 +176,27 @@ This step can be done together with step 2, but for first release you are doing
 git checkout -b osa.release.vx.testing
 ```
 
-Add vx.y target to `ci-operator/config/openshift/openshift-azure/openshift-openshift-azure-master.yaml`
+Add vx.y target to `ci-operator/jobs/openshift/openshift-azure/openshift-openshift-azure-master-presubmits.yaml`
```

Comment:
Do we have anything that removes previous versions from CI once they are no longer valid?

Reply:
Yes, it's the bottom section of this doc ("cleaning an old release").